CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ More detailed release notes can be found on the [releases page](https://github.c
 * Sequential writes to preferences does not save to file (#2449)
 * Correct Android keyboard handling (#2447)
 * MIUI-Android: The widget’s Hyperlink cannot open the URL (#1514)
+* Mobile simulator does not close (#2225)
```

Comment:
This was never an issue in earlier versions before 2.1. I’m not sure it should be in the release notes.

Reply:
Good point